common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4113,6 +4116,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(0)
 
+  val CLIENT_PUSH_CHECKSUM_HEADER_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.push.checksum.header.enabled")
```

Comment:
it's not necessary to expose implementation details in the user-oriented configuration name. storing checksum in the header, body, or tail does not matter to the functionality
```suggestion
    buildConf("celeborn.client.push.checksum.enabled")
```

Reply:
update: I noticed that configuration also affects the client read side, so `.push` might be eliminated, just `celeborn.client.shuffle.checksum.enabled`